src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -53,6 +53,7 @@
     <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.dylib" />
     <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.so" />
     <_pastShimFiles Include="clrcompression.dll" />
+    <_pastShimFiles Include="msquic.dll" />
```

Comment:
Looks to me like this isn’t needed, cc @jkoritzinsky

Reply:
I was getting 
```
 Microsoft.NETCore.App.Runtime.Composite ->
C:\Users\test\.nuget\packages\microsoft.dotnet.sharedframework.sdk\6.0.0-beta.21357.3\targets\sharedfx.targets(279,5): error : The following files are missing entries in the templated manifest: [C:\Users\test\github\wfurt-runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.Composite.sfxproj]
C:\Users\test\.nuget\packages\microsoft.dotnet.sharedframework.sdk\6.0.0-beta.21357.3\targets\sharedfx.targets(279,5): error : msquic.dll. Add these file names with extensions to the 'PlatformManifestFileEntry' item group for the runtime pack and corresponding ref pack to include them in the platform manifest. [C:\Users\test\github\wfurt-runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.Composite.sfxproj]

Build FAILED.
```
with just the copy. So I added same entry as I saw for the compression.

Reply:
The entry here on line 56 appears to be redundant.  It's appending to `_pastShimFiles` which is only used to add to `PlatformManifestFileEntry` and nowhere else.  The entry on line 165 adds the to the same `PlatformManifestFileEntry` item so you can remove the entry on 56.  Also please see the comment on line 32 which seems to be incorrect for how you are using this.

Reply:
yes, you are right. I removed the line and build still works and msquic.dll is still present in the zip file.